<launch>
    <node pkg="turtlesim" type="turtlesim_node" name="sim"/>

    <node pkg="turtlesim" type="turtle_teleop_key" name="teleop" output="screen"/>

    <node name="static_transform_publisher" pkg="tf" type="static_transform_publisher"
          args="0 0 0 0 0 0 1 map world 100"/>

    <node name="turtle1_tf_broadcaster" pkg="learning_tf" type="turtle_tf_broadcaster.py"
          respawn="false" output="screen">
      <param name="turtle" type="string" value="turtle1"/>
    </node>

    <node name="turtle2_tf_broadcaster" pkg="learning_tf" type="turtle_tf_broadcaster.py"
          respawn="false" output="screen">
      <param name="turtle" type="string" value="turtle2"/>
    </node>

    <node name="turtle_tf_listener" pkg="learning_tf" type="turtle_tf_listener.py"
          respawn="false" output="screen">
      <param name="leader_tf" type="string" value="/turtle1"/>
    </node>
  </launch>
